app/controllers/staff/organisations_controller.rb

```diff
@@ -3,6 +3,7 @@
 class Staff::OrganisationsController < Staff::BaseController
   def index
     @organisations = policy_scope(Organisation)
+    authorize @organisations
```

Comment:
Do we need to call `authorize` here? We already have the scope of organisations?